src/main/java/sa/com/cloudsolutions/antikythera/depsolver/DepSolver.java

```diff
@@ -13,14 +13,7 @@
 import com.github.javaparser.ast.body.Parameter;
 import com.github.javaparser.ast.body.TypeDeclaration;
 import com.github.javaparser.ast.body.VariableDeclarator;
-import com.github.javaparser.ast.expr.AnnotationExpr;
-import com.github.javaparser.ast.expr.BinaryExpr;
-import com.github.javaparser.ast.expr.CastExpr;
-import com.github.javaparser.ast.expr.ConditionalExpr;
-import com.github.javaparser.ast.expr.Expression;
-import com.github.javaparser.ast.expr.MethodCallExpr;
-import com.github.javaparser.ast.expr.ObjectCreationExpr;
-import com.github.javaparser.ast.expr.VariableDeclarationExpr;
+import com.github.javaparser.ast.expr.*;
```

Comment:
can we avoid the use of wildcard imports please. Having looked at the import resolution code, you know how difficult resolving wildcard imports are!
I thin these were added automatically by the IDE though. It's possible to change your intellij settings to not convert imports to wild card imports. Please try that